addons/website_sale_loyalty/views/loyalty_portal_template.xml

```diff
@@ -0,0 +1,11 @@
+<odoo>
+    <template id="webiste_sale_loyalty_button" inherit_id="loyalty.loyalty_buttons">
+        <p id="card_points" position="after">
+            <t t-if="program=='Loyalty Cards'" >
+                <t t-if="any(reward.required_points &lt;= card.points for reward in card.program_id.reward_ids)">
+                    <p class="m-0 text-info">A reward is waiting for you</p>
```

Comment:
Yes, Because the use now button in the card is shown only when website_sale is installed (which redirects to /shop)
so when website_sale is installed user can check the card and click on claim now